public/app/features/plugins/sandbox/sandbox_plugin_loader_registry.ts

```diff
@@ -64,7 +64,11 @@ export async function isPluginFrontendSandboxEligible({
   try {
     // don't run grafana-signed plugins in sandbox
     const pluginMeta = await getPluginSettings(pluginId, { showErrorAlert: false });
-    if (pluginMeta.signatureType === PluginSignatureType.grafana || pluginMeta.signature === 'internal') {
+    if (
+      pluginMeta.signatureType === PluginSignatureType.grafana ||
+      pluginMeta.signature === 'internal' ||
+      pluginId.startsWith('grafana-')
```

Comment:
curious, do we have cases where signatureType is different for `grafana-` plugins?

Reply:
not in the catalog (AFAIK) but it could be unsigned when developing

Reply:
I am not convinced with this check. A malicious plugin can potentially escape the sandbox by changing its plugin-id to something starting with `grafana-`

Reply:
plugins cannot change their id 😓 so only unsigned plugins would be able to start with `grafana-`. In any case, there is no clear use case for this (only for dev), so happy to not include it.